nodes/SuperSaaS/GenericFunctions.ts

```diff
@@ -6,6 +6,8 @@ import type {
 	IHttpRequestMethods,
 } from 'n8n-workflow';
 import { NodeApiError, NodeOperationError } from 'n8n-workflow';
+import type { AxiosError } from 'axios';
```

Comment:
Copied listing rules from the main project requires a type to the error, not 100% necessary but then the listing rule for unknown needs to be turned off.